qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -79,3 +79,21 @@
             chardev_backend_vs1 = tcp_socket
         - pty:
             chardev_backend_vs1 = pty
+    variants:
+        - @iommu_disabled:                                                              
+        - iommu_enabled:
+            virtio_dev_iommu_platform = on
+            x86_64, i386:
+                only q35
+                only unix_socket
+                only default
```

Comment:
@nanliu-r  This solution does not change the old configuration for x86, could you help confirm if x86 need cover all tests with iommu enabled ? Thanks!

Reply:
> @nanliu-r This solution does not change the old configuration for x86, could you help confirm if x86 need cover all tests with iommu enabled ? Thanks!

I think we can keep consistency with ppc.

Reply:
> 
> I think we can keep consistency with ppc.

So, we can remove `only unix_socket` and `only default` @nanliu-r  @MiriamDeng

And, i would like to highlight with the patch, the existing case  name will be changed, please be aware of this, could you help sync with virtio-win ? Thanks @nanliu-r

Reply:
I will help to remove only unix_socket and only default

Reply:
> > I think we can keep consistency with ppc.
> 
> So, we can remove `only unix_socket` and `only default` @nanliu-r @MiriamDeng
> 
> And, i would like to highlight with the patch, the existing case name will be changed, please be aware of this, could you help sync with virtio-win ? Thanks @nanliu-r

@yanan-fu Hi, after thinking again, I think it should enough for x86 now to cover on high priority case, so we won't change any thing now, and I will tell the virtio-win the change of script. thanks.

Reply:
Thanks @nanliu-r , resolve this conversation as no need to update the configuration for x86, thanks!